app/views/all_casa_admins/dashboard/show.html.erb

```diff
@@ -13,10 +13,10 @@
     </tr>
   </thead>
   <tbody>
-    <% @casa_orgs.each do |casa_org| %>
+    <% @current_organizations.each do |current_organization| %>
       <tr>
-        <td><%= link_to casa_org.name, all_casa_admins_casa_org_path(casa_org) %></td>
-        <td><%= casa_org.created_at.strftime('%B %e, %Y') %></td>
+        <td><%= link_to current_organization.name, all_casa_admins_casa_org_path(current_organization) %></td>
+        <td><%= current_organization.created_at.strftime('%B %e, %Y') %></td>
```

Comment:
The use of `current_organization` as the variable name may lead to confusion with the application level helper of the same. If one were to search for `current_organization` in the project, we'd find this reference which is a local scope variable and NOT the helper itself.

Consider simplifying by renaming the variables to `@organizations` and `organization`.

Reply:
good call 👍